From a43e4e60c8f66d7858a96d045b649d6ed2fd0c59 Mon Sep 17 00:00:00 2001
From: Bin Yang <yangbin@rock-chips.com>
Date: Tue, 8 May 2018 17:47:30 +0800
Subject: [PATCH] arm64: dts: rockchip: gpu vd always on for rk3399 mid board

Add the 'regulator-always-on' property for vdd_gpu on rk3399-tve1030g/
rk3399-tve1205g/rk3399-mid-818-android

Change-Id: I22938c7041bc56b53983447a00e83144411b4cca
Signed-off-by: Bin Yang <yangbin@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-mid-818-android.dts | 1 +
 arch/arm64/boot/dts/rockchip/rk3399-tve1030g.dts        | 1 +
 arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts        | 1 +
 3 files changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android.dts b/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android.dts
index 445ef9bb8ef0..60edaf50c5fa 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android.dts
@@ -402,6 +402,7 @@
 		regulator-max-microvolt = <1400000>;
 		regulator-ramp-delay = <1000>;
 		fcs,suspend-voltage-selector = <1>;
+		regulator-always-on;
 		regulator-boot-on;
 		regulator-state-mem {
 			regulator-off-in-suspend;
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-tve1030g.dts b/arch/arm64/boot/dts/rockchip/rk3399-tve1030g.dts
index 2a2dbb33efe9..ab2881ff9518 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-tve1030g.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-tve1030g.dts
@@ -385,6 +385,7 @@
 		regulator-max-microvolt = <1400000>;
 		regulator-ramp-delay = <1000>;
 		fcs,suspend-voltage-selector = <1>;
+		regulator-always-on;
 		regulator-boot-on;
 		regulator-state-mem {
 			regulator-off-in-suspend;
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts b/arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts
index b4002c47c792..d718eed0f63b 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts
@@ -505,6 +505,7 @@
 		regulator-max-microvolt = <1400000>;
 		regulator-ramp-delay = <1000>;
 		fcs,suspend-voltage-selector = <1>;
+		regulator-always-on;
 		regulator-boot-on;
 		regulator-state-mem {
 			regulator-off-in-suspend;
-- 
2.35.3

